extensions/amp-carousel/0.1/amp-carousel.md

```diff
@@ -268,6 +268,14 @@ _Example: Overriding the default `.amp-carousel-button` inlined SVG_
 
 Note that the SVG content needs to have certain characters, including `<`, `>` and `#` encoded. This can be done using a tool like [SVGO](https://github.com/svg/svgo) or using [`encodeURIComponent`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent).
 
+## Supported Layouts for Slides
+
+As noted above, the `type="slides"` designation on `amp-carousel 0.1` supports the following layouts: `fill`, `fixed`, `fixed-height`, `flex-item`, `nodisplay`, and `responsive`.
+
+Publishers should be aware that while this mode explictly supports `fixed` layout sizes, it also employs `display: flex;` in its styling. In other words nested elements can have `layout=fixed` sizing, though via styling it only respects absolute width up to 100%.
+
+For an accessible and smooth user experience, it is generally good practice for all children of a carousel and their parenting carousel to share the same dimensional ratios as well as the same layout type.
```

Comment:
Add "when using `type="slides"`"; while this section is about slides, it is still a bit confusing, since this is explicitly not the case for `type="carousel"`, and this sentence simply refers to the "children of a carousel".